server/maintenance_requests.go

```diff
@@ -0,0 +1,56 @@
+// This file provides NewApp() with functions for maintenance requests
+package main
+
+import(
+	"github.com/gin-gonic/gin"
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+	"time"
+	"io/ioutil"
+	"net/http"
+	"encoding/json"
+)
+
+/*type CreateRequest struct {
+	Request string `json:"request"`
+	UserSubmitted int `json:"id"`
+	DateSubmitted time.Time `json:"date"`
+}*/
+
+/* AllMaintenanceRequests returns all the maintenance_requests from the database
+*  AllMaintenanceRequests is a function of app */
+func (app *App) AllMaintenanceRequests(c *gin.Context) {
+	/* Get data using psql.AllMaintenanceRequests and pass in app.db for the
+	* connection */
+
+	/* Takes the data returned in the psql func and formats it to a valid format
+	* using the gin.H func. See if it you can just use a Struct */
+
+	/* Use the gin.context.JSON func to return a status and the data. See auth.go
+	* for an example */
+}
+
+func (app *App) CreateMaintenanceRequests(c *gin.Context) {
```

Comment:
Small thing: Let's change the name to `CreateMaintenanceRequest` since we're only creating one request at a time.

Reply:
Make sure to refactor and change the names elsewhere too.

Reply:
fix: renamed file and func